tutorial/practicals/LFRic/single_node/3_sequential/example_alg.x90

```diff
@@ -0,0 +1,16 @@
+subroutine example_code(field1, sum)
```

Comment:
Please add copyright/license and a brief comment describing the purpose of the file.

Reply:
Done.